extensions/wikia/PortableInfobox/tests/InfoboxParamsValidatorTest.php

```diff
@@ -65,4 +66,81 @@ public function testInfoboxParamsPassValidationDataProvider() {
 			]
 		];
 	}
+
+	/**
+	 * @param $color
+	 * @dataProvider testPassValidateColorValueDataProvider
+	 */
+	public function testPassValidateColorValue( $color ) {
+		$this->assertTrue( $this->InfoboxParamsValidator->validateColorValue( $color ) );
+	}
+
+	public function testPassValidateColorValueDataProvider() {
+		return [
+			['color' => 'aaa'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "color" 50 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)